Fixed Test cases not related to render

Charlyjcmu opened the pull request:
I corrected the IndexDatatype.createTypedArray by adding a checker if the number of vertices is less than 256, then there are the allocated array is Uint8array. 

I also corrected the test cases in Core/Geometry/Pipeline, Core/IndexDatatype, Scene/PrimitivePipeline, Scene/Model/GeoJsonLoader. 

I also included a test case in Core/IndexDatatype for the new code.

Charlyjcmu committed:
Change IndexDatatype and add work assignment

Charlyjcmu committed:
updated Work Schedule

Charlyjcmu committed:
Fixed not Render problems

Charlyjcmu committed:
Fixed primitivePipeline error

hredi commented:
I think the changes you've made are great, just have a few suggestions.

First, could you add yourself to the CONTRIBUTORS.md file? This file lists all of the contributors to the project, and it's a great way to give credit to everyone who has contributed.

Second, could you update the CHANGES.md file with a brief description of the changes you've made? This will help other contributors and users understand what has changed, and it will make it easier for everyone to see the progress that has been made.

Charlyjcmu committed:
Added Contributors.md file

Charlyjcmu commented:
I have added our names to the contributors.md file and changed the file WireframPipelineStage.js to no longer check for UintArrays. I don't think we need to change the Changes.md file.

hredi reviewed (approved):
Good work!
